esmvalcore/cmor/check.py

```diff
@@ -283,6 +284,21 @@ def _check_rank(self):
             self.report_error(self._does_msg, self._cube.var_name,
                               'match coordinate rank')
 
+    def _check_multiple_coords_same_stdname(self):
+        standard_names = set()
+        for coord in self._cube.coords():
+            if coord.standard_name:
+                if coord.standard_name in standard_names:
+                    coords = [c.var_name for c in self._cube.coords(
+                        standard_name=coord.standard_name)
+                    ]
```

Comment:
any way we can fix this on the fly and report a warning rather than just collecting the duplicates and reporting an error?